recipe/run_conda_forge_build_setup_linux

```diff
@@ -1,9 +1,5 @@
 #!/bin/bash
 
-# 2 cores are available on CircleCI workers: https://discuss.circleci.com/t/what-runs-on-the-node-container-by-default/1443
-# MAKEFLAGS is passed through conda build: https://github.com/conda/conda-build/pull/917
-export MAKEFLAGS="-j2 ${MAKEFLAGS}"
```

Comment:
Not that this is in a `conda-build` release yet, but just to get this ready for it. Can we please restore the comment above, newline below and change the `export` statement to `export CPU_COUNT=2`?